quodlibet/quodlibet/qltk/lyrics.py

```diff
@@ -113,23 +113,45 @@ def __search(self, song, buffer, refresh, add):
             GLib.idle_add(buffer.set_text, text)
             GLib.idle_add(refresh.set_sensitive, True)
 
-    def __save(self, save, lyricname, buffer, delete):
+    def __save(self, save, song, buffer, delete):
+        start, end = buffer.get_bounds()
+        text = buffer.get_text(start, end, True)
+
+        # First, write back to the tags.
+        song["lyrics"] = text
```

Comment:
Done. So you mean that the text in the buffer is in the system's global encoding which is not necessarily utf-8?

If yes, then we also need to convert from utf-8 to system encoding on load — how to do that? (/me is not familiar with Python at all, this is probably my first python patch ever)